examples/ui-demo/src/app/providers.tsx

```diff
@@ -15,18 +18,24 @@ export const Providers = (
     initialConfig?: Config;
   }>
 ) => {
-  const localAlchemyConfig = {
-    ...alchemyConfig,
```

Comment:
(Very nit) This is fine, but you wouldn't need the IIFE if you just changed this line to `...alchemyConfig(),`.

Reply:
oh we also have this below that depends on the result of `alchemyConfig()`
```ts
ui: props.initialConfig
          ? convertDemoConfigToUiConfig(props.initialConfig)
          : innerConfig.ui
```